2016/05/10 23:09:39 [error] [exception.CException] exception 'CException' with message 'Alias "bootstrap.components.Bootstrap" is invalid. Make sure it points to an existing directory or file.' in C:\xampp\htdocs\web\framework\YiiBase.php:348
Stack trace:
#0 C:\xampp\htdocs\web\framework\YiiBase.php(196): YiiBase::import('bootstrap.compo...', true)
#1 C:\xampp\htdocs\web\framework\base\CModule.php(393): YiiBase::createComponent(Array)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('bootstrap')
#3 C:\xampp\htdocs\web\protected\views\layouts\main.php(19): CModule->__get('bootstrap')
#4 C:\xampp\htdocs\web\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\web\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\web\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\web\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#8 C:\xampp\htdocs\web\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#9 C:\xampp\htdocs\web\framework\web\CBaseController.php(214): COutputProcessor->run()
#10 C:\xampp\htdocs\web\framework\web\CBaseController.php(308): CBaseController->endWidget('CContentDecorat...')
#11 C:\xampp\htdocs\web\protected\views\layouts\column1.php(6): CBaseController->endContent()
#12 C:\xampp\htdocs\web\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#13 C:\xampp\htdocs\web\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#14 C:\xampp\htdocs\web\framework\web\CController.php(784): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#15 C:\xampp\htdocs\web\protected\controllers\SiteController.php(32): CController->render('index')
#16 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#17 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#18 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#19 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('')
#21 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#22 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#23 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#24 {main}
REQUEST_URI=/web/index.php
---
2016/05/10 23:15:06 [error] [exception.CException] exception 'CException' with message 'Alias "bootstrap.components.Bootstrap" is invalid. Make sure it points to an existing directory or file.' in C:\xampp\htdocs\web\framework\YiiBase.php:348
Stack trace:
#0 C:\xampp\htdocs\web\framework\YiiBase.php(196): YiiBase::import('bootstrap.compo...', true)
#1 C:\xampp\htdocs\web\framework\base\CModule.php(393): YiiBase::createComponent(Array)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('bootstrap')
#3 C:\xampp\htdocs\web\protected\views\layouts\main.php(19): CModule->__get('bootstrap')
#4 C:\xampp\htdocs\web\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\web\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\web\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\web\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#8 C:\xampp\htdocs\web\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#9 C:\xampp\htdocs\web\framework\web\CBaseController.php(214): COutputProcessor->run()
#10 C:\xampp\htdocs\web\framework\web\CBaseController.php(308): CBaseController->endWidget('CContentDecorat...')
#11 C:\xampp\htdocs\web\protected\views\layouts\column1.php(6): CBaseController->endContent()
#12 C:\xampp\htdocs\web\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#13 C:\xampp\htdocs\web\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#14 C:\xampp\htdocs\web\framework\web\CController.php(784): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#15 C:\xampp\htdocs\web\protected\controllers\SiteController.php(32): CController->render('index')
#16 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#17 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#18 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#19 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('')
#21 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#22 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#23 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#24 {main}
REQUEST_URI=/web/index.php
---
2016/05/10 23:16:04 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'web'@'localhost' (using password: YES)' in C:\xampp\htdocs\web\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\web\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\web\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\xampp\htdocs\web\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/web/index.php?r=gii
---
2016/05/10 23:16:43 [error] [exception.CException] exception 'CException' with message 'Alias "bootstrap.components.Bootstrap" is invalid. Make sure it points to an existing directory or file.' in C:\xampp\htdocs\web\framework\YiiBase.php:348
Stack trace:
#0 C:\xampp\htdocs\web\framework\YiiBase.php(196): YiiBase::import('bootstrap.compo...', true)
#1 C:\xampp\htdocs\web\framework\base\CModule.php(393): YiiBase::createComponent(Array)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('bootstrap')
#3 C:\xampp\htdocs\web\protected\views\layouts\main.php(19): CModule->__get('bootstrap')
#4 C:\xampp\htdocs\web\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\web\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\web\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\web\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#8 C:\xampp\htdocs\web\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#9 C:\xampp\htdocs\web\framework\web\CBaseController.php(214): COutputProcessor->run()
#10 C:\xampp\htdocs\web\framework\web\CBaseController.php(308): CBaseController->endWidget('CContentDecorat...')
#11 C:\xampp\htdocs\web\protected\views\layouts\column1.php(6): CBaseController->endContent()
#12 C:\xampp\htdocs\web\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#13 C:\xampp\htdocs\web\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#14 C:\xampp\htdocs\web\framework\web\CController.php(784): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#15 C:\xampp\htdocs\web\framework\web\actions\CViewAction.php(140): CController->render('pages/about')
#16 C:\xampp\htdocs\web\framework\web\actions\CAction.php(76): CViewAction->run()
#17 C:\xampp\htdocs\web\framework\web\CController.php(308): CAction->runWithParams(Array)
#18 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#19 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#20 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('page')
#21 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#22 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#23 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#24 {main}
REQUEST_URI=/web/index.php?r=site/page&view=about
HTTP_REFERER=http://localhost/web/index.php?r=site/contact
---
2016/05/10 23:31:11 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'web'@'localhost' (using password: YES)' in C:\xampp\htdocs\web\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\web\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\web\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\xampp\htdocs\web\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/web/index.php?r=gii
---
2016/05/10 23:31:14 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'web'@'localhost' (using password: YES)' in C:\xampp\htdocs\web\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\web\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\web\framework\gii\generators\crud\CrudCode.php(44): CModule->__get('db')
#5 C:\xampp\htdocs\web\framework\web\CFormModel.php(39): CrudCode->init()
#6 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud/index')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/index.php?r=gii/crud/index
HTTP_REFERER=http://localhost/web/index.php?r=gii
---
2016/05/10 23:31:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'web'@'localhost' (using password: YES)' in C:\xampp\htdocs\web\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\web\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\web\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\xampp\htdocs\web\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/web/index.php?r=gii
---
2016/05/10 23:32:15 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'web'@'localhost' (using password: YES)' in C:\xampp\htdocs\web\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\web\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\web\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\xampp\htdocs\web\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/web/index.php?r=gii
---
2016/05/10 23:39:00 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'web'@'localhost' (using password: YES)' in C:\xampp\htdocs\web\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\web\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\web\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\xampp\htdocs\web\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/web/index.php?r=gii
---
2016/05/10 23:39:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gi".' in C:\xampp\htdocs\web\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('gi')
#1 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/web/index.php?r=gi
---
2016/05/10 23:39:10 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'web'@'localhost' (using password: YES)' in C:\xampp\htdocs\web\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\web\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\web\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\xampp\htdocs\web\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/web/index.php?r=gii
---
2016/05/11 00:02:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "ersror".' in C:\xampp\htdocs\web\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CController.php(270): CController->missingAction('ersror')
#1 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('ersror')
#2 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('site/ersror')
#3 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/web/index.php?r=site/ersror
---
2016/05/11 00:02:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "efrror".' in C:\xampp\htdocs\web\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CController.php(270): CController->missingAction('efrror')
#1 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('efrror')
#2 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('site/efrror')
#3 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/web/index.php?r=site/efrror
---
2016/05/11 00:02:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "sefse".' in C:\xampp\htdocs\web\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CController.php(270): CController->missingAction('sefse')
#1 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('sefse')
#2 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('site/sefse')
#3 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/web/index.php?r=site/sefse
---
2016/05/11 00:05:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "_view".' in C:\xampp\htdocs\web\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CController.php(270): CController->missingAction('_view')
#1 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('_view')
#2 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('cliente/_view')
#3 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/web/index.php?r=cliente/_view
---
2016/05/11 00:05:21 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\web\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('cliente/update')
#10 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/web/index.php?r=cliente/update
---
2016/05/11 00:05:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "_view.php".' in C:\xampp\htdocs\web\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CController.php(270): CController->missingAction('_view.php')
#1 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('_view.php')
#2 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('cliente/_view.p...')
#3 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/web/index.php?r=cliente/_view.php
---
2016/05/11 00:05:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\xampp\htdocs\web\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('cliente/index.p...')
#3 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/web/index.php?r=cliente/index.php
---
2016/05/11 21:18:38 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('cliente/admin')
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/web/index.php?r=cliente/admin
HTTP_REFERER=http://localhost/web/index.php?r=cliente/index
---
2016/05/11 22:15:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "about".' in C:\xampp\htdocs\web\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CController.php(270): CController->missingAction('about')
#1 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('about')
#2 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('site/about')
#3 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/web/index.php/site/about
HTTP_REFERER=http://localhost/web/index.php/site/contact
---
2016/05/11 22:21:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "handicap/index".' in C:\xampp\htdocs\web\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('handicap/index')
#1 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/web/index.php/handicap/index
HTTP_REFERER=http://localhost/web/index.php/estado/index
---
2016/05/11 23:33:53 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in C:\xampp\htdocs\web\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('cita/update')
#10 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/web/index.php/cita/update
HTTP_REFERER=http://localhost/web/index.php/site/login
---
2016/05/12 21:59:05 [error] [php] Undefined variable: model (C:\xampp\htdocs\web\protected\views\layouts\front.php:6)
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CController.php(784): CalendarioController->renderFile()
#1 C:\xampp\htdocs\web\protected\controllers\CalendarioController.php(10): CalendarioController->render()
#2 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CalendarioController->actionIndex()
#3 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 C:\xampp\htdocs\web\framework\web\CController.php(286): CalendarioController->runAction()
#5 C:\xampp\htdocs\web\framework\web\CController.php(265): CalendarioController->runActionWithFilters()
#6 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CalendarioController->run()
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\web\index.php(13): CWebApplication->run()
REQUEST_URI=/web/index.php/calendario/index
in C:\xampp\htdocs\web\protected\views\layouts\front.php (6)
in C:\xampp\htdocs\web\protected\controllers\CalendarioController.php (10)
in C:\xampp\htdocs\web\index.php (13)
2016/05/12 22:00:33 [error] [php] Undefined variable: model (C:\xampp\htdocs\web\protected\views\layouts\front.php:6)
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CController.php(784): CalendarioController->renderFile()
#1 C:\xampp\htdocs\web\protected\controllers\CalendarioController.php(10): CalendarioController->render()
#2 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CalendarioController->actionIndex()
#3 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 C:\xampp\htdocs\web\framework\web\CController.php(286): CalendarioController->runAction()
#5 C:\xampp\htdocs\web\framework\web\CController.php(265): CalendarioController->runActionWithFilters()
#6 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CalendarioController->run()
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\web\index.php(13): CWebApplication->run()
REQUEST_URI=/web/index.php/calendario/index
in C:\xampp\htdocs\web\protected\views\layouts\front.php (6)
in C:\xampp\htdocs\web\protected\controllers\CalendarioController.php (10)
in C:\xampp\htdocs\web\index.php (13)
2016/05/12 22:01:00 [error] [exception.CException] exception 'CException' with message 'CalendarioController cannot find the requested view "_tab_generales".' in C:\xampp\htdocs\web\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\widgets\CTabView.php(217): CController->renderPartial('_tab_generales', '$model')
#1 C:\xampp\htdocs\web\framework\web\widgets\CTabView.php(149): CTabView->renderBody()
#2 C:\xampp\htdocs\web\framework\web\CBaseController.php(181): CTabView->run()
#3 C:\xampp\htdocs\web\protected\views\layouts\front.php(22): CBaseController->widget('CTabView', Array)
#4 C:\xampp\htdocs\web\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\web\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\web\framework\web\CController.php(784): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\web\protected\controllers\CalendarioController.php(10): CController->render('index')
#8 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CalendarioController->actionIndex()
#9 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('calendario/inde...')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/index.php/calendario/index
---
2016/05/14 14:18:34 [error] [php] Object of class CInlineAction could not be converted to string (C:\xampp\htdocs\web\protected\views\calendario\index.php:13)
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CController.php(869): CalendarioController->renderFile()
#1 C:\xampp\htdocs\web\framework\web\CController.php(782): CalendarioController->renderPartial()
#2 C:\xampp\htdocs\web\protected\controllers\CalendarioController.php(10): CalendarioController->render()
#3 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CalendarioController->actionIndex()
#4 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\web\framework\web\CController.php(286): CalendarioController->runAction()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CalendarioController->runActionWithFilters()
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CalendarioController->run()
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CWebApplication->run()
REQUEST_URI=/web/index.php/calendario/index
in C:\xampp\htdocs\web\protected\views\calendario\index.php (13)
in C:\xampp\htdocs\web\protected\controllers\CalendarioController.php (10)
in C:\xampp\htdocs\web\index.php (13)
2016/05/14 14:18:53 [error] [php] Object of class CInlineAction could not be converted to string (C:\xampp\htdocs\web\protected\views\calendario\index.php:13)
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CController.php(869): CalendarioController->renderFile()
#1 C:\xampp\htdocs\web\framework\web\CController.php(782): CalendarioController->renderPartial()
#2 C:\xampp\htdocs\web\protected\controllers\CalendarioController.php(10): CalendarioController->render()
#3 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CalendarioController->actionIndex()
#4 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\web\framework\web\CController.php(286): CalendarioController->runAction()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CalendarioController->runActionWithFilters()
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CalendarioController->run()
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CWebApplication->run()
REQUEST_URI=/web/index.php/calendario/index
in C:\xampp\htdocs\web\protected\views\calendario\index.php (13)
in C:\xampp\htdocs\web\protected\controllers\CalendarioController.php (10)
in C:\xampp\htdocs\web\index.php (13)
2016/05/14 14:19:10 [error] [php] Object of class CInlineAction could not be converted to string (C:\xampp\htdocs\web\protected\views\calendario\index.php:13)
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CController.php(869): CalendarioController->renderFile()
#1 C:\xampp\htdocs\web\framework\web\CController.php(782): CalendarioController->renderPartial()
#2 C:\xampp\htdocs\web\protected\controllers\CalendarioController.php(10): CalendarioController->render()
#3 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CalendarioController->actionIndex()
#4 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\web\framework\web\CController.php(286): CalendarioController->runAction()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CalendarioController->runActionWithFilters()
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CalendarioController->run()
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CWebApplication->run()
REQUEST_URI=/web/index.php/calendario/index
in C:\xampp\htdocs\web\protected\views\calendario\index.php (13)
in C:\xampp\htdocs\web\protected\controllers\CalendarioController.php (10)
in C:\xampp\htdocs\web\index.php (13)
2016/05/14 14:19:21 [error] [php] Object of class CInlineAction could not be converted to string (C:\xampp\htdocs\web\protected\views\calendario\index.php:13)
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CController.php(869): CalendarioController->renderFile()
#1 C:\xampp\htdocs\web\framework\web\CController.php(782): CalendarioController->renderPartial()
#2 C:\xampp\htdocs\web\protected\controllers\CalendarioController.php(10): CalendarioController->render()
#3 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CalendarioController->actionIndex()
#4 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\web\framework\web\CController.php(286): CalendarioController->runAction()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CalendarioController->runActionWithFilters()
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CalendarioController->run()
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CWebApplication->run()
REQUEST_URI=/web/index.php/calendario/index
in C:\xampp\htdocs\web\protected\views\calendario\index.php (13)
in C:\xampp\htdocs\web\protected\controllers\CalendarioController.php (10)
in C:\xampp\htdocs\web\index.php (13)
2016/05/14 14:19:41 [error] [php] Object of class CInlineAction could not be converted to string (C:\xampp\htdocs\web\protected\views\calendario\index.php:13)
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CController.php(869): CalendarioController->renderFile()
#1 C:\xampp\htdocs\web\framework\web\CController.php(782): CalendarioController->renderPartial()
#2 C:\xampp\htdocs\web\protected\controllers\CalendarioController.php(10): CalendarioController->render()
#3 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CalendarioController->actionIndex()
#4 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\web\framework\web\CController.php(286): CalendarioController->runAction()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): CalendarioController->runActionWithFilters()
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CalendarioController->run()
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CWebApplication->run()
REQUEST_URI=/web/index.php/calendario/index
in C:\xampp\htdocs\web\protected\views\calendario\index.php (13)
in C:\xampp\htdocs\web\protected\controllers\CalendarioController.php (10)
in C:\xampp\htdocs\web\index.php (13)
2016/05/14 16:49:26 [error] [php] Undefined variable: model (C:\xampp\htdocs\web\protected\views\registro\index.php:35)
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CController.php(869): RegistroController->renderFile()
#1 C:\xampp\htdocs\web\framework\web\CController.php(782): RegistroController->renderPartial()
#2 C:\xampp\htdocs\web\protected\controllers\RegistroController.php(10): RegistroController->render()
#3 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): RegistroController->actionIndex()
#4 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\web\framework\web\CController.php(286): RegistroController->runAction()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): RegistroController->runActionWithFilters()
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): RegistroController->run()
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CWebApplication->run()
REQUEST_URI=/web/index.php/registro/index
in C:\xampp\htdocs\web\protected\views\registro\index.php (35)
in C:\xampp\htdocs\web\protected\controllers\RegistroController.php (10)
in C:\xampp\htdocs\web\index.php (13)
2016/05/14 16:51:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\xampp\htdocs\web\protected\controllers\ClienteController.php:157
Stack trace:
#0 C:\xampp\htdocs\web\protected\controllers\ClienteController.php(89): ClienteController->loadModel('2')
#1 [internal function]: ClienteController->actionUpdate('2')
#2 C:\xampp\htdocs\web\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(ClienteController), Array)
#3 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(ClienteController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\web\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\web\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\web\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\web\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\web\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('update')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('cliente/update')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/index.php/cliente/update/2
---
2016/05/14 17:04:28 [error] [php] Undefined variable: model (C:\xampp\htdocs\web\protected\views\registro\index.php:35)
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CController.php(869): RegistroController->renderFile()
#1 C:\xampp\htdocs\web\framework\web\CController.php(782): RegistroController->renderPartial()
#2 C:\xampp\htdocs\web\protected\controllers\RegistroController.php(10): RegistroController->render()
#3 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): RegistroController->actionIndex()
#4 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\web\framework\web\CController.php(286): RegistroController->runAction()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): RegistroController->runActionWithFilters()
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): RegistroController->run()
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CWebApplication->run()
REQUEST_URI=/web/index.php/registro
in C:\xampp\htdocs\web\protected\views\registro\index.php (35)
in C:\xampp\htdocs\web\protected\controllers\RegistroController.php (10)
in C:\xampp\htdocs\web\index.php (13)
2016/05/14 17:05:20 [error] [php] Undefined variable: form (C:\xampp\htdocs\web\protected\views\registro\index.php:25)
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CController.php(869): RegistroController->renderFile()
#1 C:\xampp\htdocs\web\framework\web\CController.php(782): RegistroController->renderPartial()
#2 C:\xampp\htdocs\web\protected\controllers\RegistroController.php(10): RegistroController->render()
#3 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): RegistroController->actionIndex()
#4 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\web\framework\web\CController.php(286): RegistroController->runAction()
#6 C:\xampp\htdocs\web\framework\web\CController.php(265): RegistroController->runActionWithFilters()
#7 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): RegistroController->run()
#8 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp\htdocs\web\index.php(13): CWebApplication->run()
REQUEST_URI=/web/index.php/registro
in C:\xampp\htdocs\web\protected\views\registro\index.php (25)
in C:\xampp\htdocs\web\protected\controllers\RegistroController.php (10)
in C:\xampp\htdocs\web\index.php (13)
